.github/workflows/nodejs.yml

```diff
@@ -17,10 +17,11 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - name: npm install, build, and test
+    - name: npm install, build, test and lint
       run: |
         npm install
         npm run build --if-present
         npm test
+        npm run lint
```

Comment:
:+1: Wouldn't it be better to lint before running the test?

Reply:
Does it matter? Both must run.
But if you wish that I change the order, then I'll do it :-)

Reply:
> Does it matter? Both must run.

I've always thought that this is the right order. First you run "cheaper" lint and fail early if there are errors, and only then run more expensive tests. 

But it's fine, the codebase is not that big :)  
Was just wondering

Reply:
This is one way of defining the order.
I thought that a proper working test is more important than a proper lint. Therefore, I test the critical stuff first.

Reply:
Agree. As long as both run :+1: